code/modules/maps/spawners/spawners.dm

```diff
@@ -1000,3 +1000,127 @@
+		/obj/item/clothing/accessory/medal/gold,
+		/obj/item/clothing/accessory/medal/gold/captain,
+		/obj/item/clothing/accessory/medal/gold/heroism,
+		/obj/item/weapon/reagent_containers/food/snacks/chocolatebar/wrapped,
+		/obj/item/weapon/reagent_containers/food/snacks/no_raisin,
+		/obj/item/weapon/bikehorn,
+		/obj/item/weapon/storage/box/emps,
+		/obj/item/weapon/gun/projectile/automatic/mini_uzi,
+		/obj/item/clothing/suit/armor/laserproof,
+		/obj/item/weapon/reagent_containers/glass/bottle/frostoil,
+		/obj/item/weapon/reagent_containers/glass/bottle/chloralhydrate,
+		/obj/item/weapon/storage/pill_bottle/nanobot,
+		/obj/item/weapon/card/id/captains_spare,
+		/obj/item/weapon/melee/energy/axe/rusty,
+		/obj/item/voucher/free_item,
+		/obj/item/clothing/accessory/holster/,
+		/obj/item/weapon/gun/projectile/russian,
+		/obj/item/weapon/gun/mahoguny,
+		/obj/item/weapon/gun/stickybomb,
+		/obj/item/clothing/head/helmet/siren,
+		/obj/item/clothing/glasses/scanner/night,
+		/obj/item/clothing/head/collectable/petehat,
+		/obj/item/clothing/head/helmet/tactical/HoS/dermal,
+		/obj/item/clothing/under/chameleon,
+		/obj/item/clothing/gloves/anchor_arms
+		)
+
+/obj/map/spawner/safe/medical
+	name = "safe medical spawner"
+	icon_state = "safe_medical"
+	to_spawn = list(/obj/item/weapon/storage/pill_bottle/creatine,
+	/obj/item/weapon/storage/firstaid/adv,
+	/obj/item/weapon/reagent_containers/glass/bottle/random,
+	/obj/item/weapon/reagent_containers/glass/bottle/frostoil,
+	/obj/item/weapon/reagent_containers/glass/bottle/chloralhydrate,
+	/obj/item/weapon/storage/pill_bottle/nanobot,
+	/obj/item/voucher/free_item
+)
+
+
+/obj/map/spawner/safe/food
+	name = "safe food spawner"
+	icon_state = "safe_food"
+	to_spawn = list(/obj/item/weapon/reagent_containers/food/drinks/bottle/vodka,
+	/obj/item/weapon/reagent_containers/food/snacks/chocolatebar/wrapped,
+	/obj/item/weapon/reagent_containers/food/snacks/no_raisin,
+	/obj/item/voucher/free_item,
+	/obj/item/voucher/free_item/hot_drink
+)
+
+/obj/map/spawner/safe/weapon
+	name = "safe weapon spawner"
+	icon_state = "safe_weapon"
+	to_spawn = list(/obj/item/weapon/shield/energy,
+	/obj/item/weapon/gun/energy/gun/nuclear,
+	/obj/item/weapon/gun/projectile/mateba,
+	/obj/item/weapon/gun/projectile/deagle/gold,
+	/obj/item/weapon/bikehorn,
+	/obj/item/weapon/storage/box/emps,
+	/obj/item/weapon/gun/projectile/automatic/mini_uzi,
+	/obj/item/weapon/melee/energy/axe/rusty,
+	/obj/item/voucher/free_item,
+	/obj/item/weapon/gun/projectile/russian,
+	/obj/item/weapon/gun/mahoguny,
+	/obj/item/weapon/gun/stickybomb,
+	/obj/item/weapon/gun/siren
+)
+
+/obj/map/spawner/safe/clothing
+	name = "safe clothing spawner"
+	icon_state = "safe_clothing"
+	to_spawn = list(/obj/item/weapon/shield/energy,
+	/obj/item/clothing/accessory/storage/webbing,
+	/obj/item/clothing/under/sexyclown,
+	/obj/item/clothing/suit/armor/laserproof,
+	/obj/item/voucher/free_item,
+	/obj/item/clothing/accessory/holster/,
```

Comment:
Oh I definitely did before even replying.  I don't know what would have happened if I forced it to spawn but I'm guessing something not good.